pkg/ccl/backupccl/restore_online.go

```diff
@@ -47,6 +48,12 @@ var onlineRestoreLinkWorkers = settings.RegisterByteSizeSetting(
 	settings.PositiveInt,
 )
 
+const downloadJobDescriptionPrefix = "Background Data Download"
+
+func (r *restoreResumer) isDownloadJob() bool {
+	return strings.Contains(r.job.Payload().Description, downloadJobDescriptionPrefix)
```

Comment:
I can't come up with a good reason to dislike this, but I do. These strings are supposed to be for and only for humans. I'd rather add a new bool to the proto than use this?

Reply:
fine by me. happy to add another bool to the proto

Reply:
Done.